src/SortableList.js

```diff
@@ -154,16 +155,27 @@ export default class SortableList extends Component {
     const {contentHeight, contentWidth, scrollEnabled} = this.state;
     const containerStyle = StyleSheet.flatten([this.props.style, this.state.style]);
     const innerContainerStyle = [styles.container];
+    let refreshControl = this.props.refreshControl;
 
     if (horizontal) {
       innerContainerStyle.push({width: contentWidth});
     } else {
       innerContainerStyle.push({height: contentHeight});
     }
 
+    if(refreshControl) {
+      refreshControl=(
+        <RefreshControl
```

Comment:
can we pass any element as refreshControl to the ScrollView. if so we can replace element from user with RefreshControl from react native